sdk/device/device_specs.py

```diff
@@ -0,0 +1,22 @@
+import json
+from typing import Any, Dict, List
+
+from pydantic import BaseModel, validator
+
+from sdk.device.device_types import DeviceType
+
+
+class DeviceSpecs(BaseModel):
+
+    device_type: DeviceType
+    specs: Dict[str, Any]
+
+    @validator("specs", pre=True)
+    def _convert_specs_to_dict(cls, v: str) -> Dict[str, Any]:
+        specs_dict: Dict[str, Any] = json.loads(v)
+        return specs_dict
+
+
+class DeviceSpecsList(BaseModel):
+
+    device_specs_list: List[DeviceSpecs]
```

Comment:
I'm unsure about the usage of this, so this might not be relevant, but would it not make sense to be able to get the names of the devices easily?

Reply:
In the current implementation, specs are defined for a given device type (not per device). The device_type is included in the DeviceSpecs so the information is available. For convenience we could maybe expose a dictionnary instead of a list though, what do you guys think ?

Reply:
But I thought part of the interest here was to really have this info at a per device level?

Not knowing/understanding usage entirely I don't have an opinion about what's the better structure

Reply:
@awennersteen to summarise:

- device specs are expected to be shared accross device_types (two fresnel devices should have the same specs)
- calibrated layouts should be individual to a device
- we are now, and for the foreseeable future not expecting to have more than one device per device_type (Fresnel)
- we've therefore attached the calibrated layouts to the device_specs as well

In the future we might need to distinguish between devices but they would still share their specs, only the calibrated layouts should vary.

Reply:
thanks